metricshub-doc/src/site/markdown/usecases/ping.md

```diff
@@ -0,0 +1,65 @@
+keywords: ping
+description: How to ping resources with MetricsHub
+
+# Pinging resources
+
+## Overview
+
+With **MetricsHub**, you can ping your resources through ICMP (Internet Control Message Protocol) to verify that they can be reached within an acceptable response time. To achieve this, you just need to configure in the `config/metricshub.yaml` file:
+* the resources to be monitored 
+* the ICMP Ping protocol for each of your resource.
+
+In the example below, we ping 9 resources that belong to the  `baie9` resource group using the **ICMP protocol**. A timeout of 3 seconds is configured and the result (OK, KO status) is displayed in a Grafana Dashboard.
```

Comment:
`baie9` is very French. Please use an English name.

Actually, there is no need to use a *resourceGroup* here, so let's avoid that and declare the multi-host resource straight.